src/components/modal/v2/parts/views/ShortTerm/styles.scss

```
@@ -145,6 +145,11 @@
     }
 
     .disclosure {
+        &.v5Design {
+            color: $v5-gray;
+            padding-top: 24px;
```

```suggestion
            padding-top: 24px;
            margin-top: auto !important;
```
Just to keep the disclosure pinned to the bottom.